### PR TITLE
Devops-83 Destroy Lisboa Stage

### PR DESCRIPTION
**Pull request by mikhaelsantos**

### CODE REVIEW

File: app/src/editor/canvas/components/Toolbar.pcss [hunk 1]
```diff
@@ -396,6 +396,7 @@ body .RealtimeRunButtonMenu {
 .CalendarWrapper {
   bottom: 50px;
   left: 50%;
+  min-height: 361px; /* Height of 6 rows/weeks. */
```
Comment (juhah) [inline app/src/editor/canvas/components/Toolbar.pcss]: These are probably unrelated?

**Comment by mikhaelsantos** [comment]
@mondoreale these changes are unrelated, please remove.

**Comment by mondoreale** [comment]
@mikhaelsantos not a problem, rebased it on current `development`.

**Comment by mondoreale** [comment]
@mikhaelsantos, for the future, in order to avoid this type of confusion, please start feature branches at `origin/development` (unless intentions dictate otherwise). Looks like your feature branch started where my https://github.com/streamr-dev/streamr-platform/pull/508 left off.

**Pull request merged**